miqi/runtime/tool_registry_factory.py

```
@@ -217,9 +217,20 @@ def create_runtime_tool_registry(
     #    exposure; execution blocks on the shared user-input gate so the
     #    desktop can render the confirm card (legacy bridge path).
     from miqi.agent.tools.ask_user_confirm import AskUserConfirmCardTool
+    from miqi.agent.tools.ask_user_plan_confirm import AskUserPlanConfirmTool
+    from miqi.agent.tools.request_action_confirmation import RequestActionConfirmationTool
     from miqi.agent.user_input_resolver import make_resolver
 
     registry.register(AskUserConfirmCardTool(resolver=make_resolver()))
+    # v3.3: todo_write（Agent 进度状态协议）——运行时由 turn_runner 用
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)